common/.commonfiles.sha

```diff
@@ -1 +1 @@
-45c6eb44b33b5df885719d4d5239b2038214852d
+9da77cfa03da2f1abdb2c325b63b6e68952c785f
```

Comment:
@jwendell this commit comes from release 1.16. Is it a problem? We can also cherry-pick the same fix for 1.15, but not for 1.14.

Reply:
AFAICT this should not be an issue. This SHA is merely for informational purposes. It's not used anywhere, differently than the proxy sha in istio.deps. The worse it can happen is a merge conflict when pulling in upstream changes into 2.3 in the future. But this is very unlikely to happen because upstream 1.14 is dead, right?

Reply:
> because upstream 1.14 is dead, right?

Yes, it is.